public/scripts/timeline.js

```diff
@@ -1597,6 +1608,22 @@ function isValidDate(dateString)
     return day > 0 && day <= monthLength[month];
 };
 
+function generateMementoURIList(object)
+{
+    var URIM = [];
+
+    $.each(object,function(index,obj){
+        if($(obj.event_html).attr("src").indexOf("notcaptured") < 0){
+            URIM[index] = obj.event_link;
+        }
+    });
+    var download = document.getElementById("downloadMementoURI");
+    download.href = "";
+    var data = URIM.toString();

```

Comment:
You can join array entries on `\n` to eliminate the need of replacing commas with new lines later:

```suggestion
    var data = URIM.join('\n');
```